spec/services/participants/change_schedule_spec.rb

```diff
@@ -56,14 +58,9 @@
       end
 
       context "when the schedule identifier change of the same type again" do
-        before do
-          create(:schedule, :npq_leadership_autumn, cohort:)
-        end
+        let(:new_schedule) { create(:schedule, :npq_leadership_spring, cohort:) }
 
-        it "is invalid and returns an error message" do
-          expect(subject.change_schedule).to be_truthy
-          expect(subject).to have_error(:schedule_identifier, :already_on_the_profile, "Selected schedule is already on the profile")
-        end
+        it { is_expected.to have_error(:schedule_identifier, :already_on_the_profile, "Selected schedule is already on the profile") }
```

Comment:
any reason we changed this test? as it was testing attempting the change twice? we could set up a new contract for that course?

Reply:
This spec was failing after the changes but I realised it was kind confusing as well with the first expect to be truthy and the second expecting an error:

`expect(subject.change_schedule).to be_truthy`
`expect(subject).to have_error(:schedule_identifier, :already_on_the_profile, "Selected schedule is already on the profile")`

Make the changes to keep it in line with the test above as well.